exec/java-exec/src/main/java/org/apache/drill/exec/util/MemoryAllocationUtilities.java

```diff
@@ -138,16 +139,36 @@ public static long computeOperatorMemory(OptionSet optionManager, long maxAllocP
   @VisibleForTesting
   public static long computeQueryMemory(DrillConfig config, OptionSet optionManager, long directMemory) {
 
+    // Get the options
+    double percent_per_query = optionManager.getOption(ExecConstants.PERCENT_MEMORY_PER_QUERY);
```

Comment:
Minor nitpick, but the convention in Java is to use Camel case for names

`percentPerQuery`

Reply:
Fixed ....